JS/Lesson35/tugba/my-app/src/components/RecipeList/index.jsx

```diff
@@ -1,18 +1,29 @@
-import { Link } from 'react-router-dom';
+import React from 'react';
+import { List, ListItem, Typography } from '@mui/material';
+import { Link, useNavigate } from 'react-router-dom';
 
-export const RecipeList = ({ recipes }) => {
-	return (
-		<div>
-			<h2>Search Results</h2>
-			<ul>
-				{recipes.map((recipe) => {
-					return (
-						<li key={recipe.idMeal}>
-							<Link to={`recipes/${recipe.idMeal}`}>{recipe.strMeal}</Link>
-						</li>
-					);
-				})}
-			</ul>
-		</div>
-	);
+const RecipeList = ({ recipes }) => {
+  const navigate = useNavigate();
+
+  const onRecipeClick = (recipe) => {
+    navigate(`/recipe/${recipe.id}`);
+    console.log(`Clicked on recipe: ${recipe.strMeal}`);
+  };
+
+  return (
+    <div>
+      <Typography variant="h5">Search Results</Typography>
+      <List>
+        {recipes.map((recipe) => (
+          <ListItem key={recipe.id} button onClick={() => onRecipeClick(recipe)}>
+            <Link to={`/recipe/${recipe.id}`} style={{ textDecoration: 'none', color: 'inherit' }}>
```

Comment:
Correct name of id used in API is `idMeal` so that's the name you need to use everywhere if you want to use id of a meal, otherwise code does not work.
```suggestion
            <Link to={`/recipe/${recipe.idMeal}`} style={{ textDecoration: 'none', color: 'inherit' }}>
```